### PR TITLE
Fix/multiplayer sync

### PR DESCRIPTION
**Pull request by andersfylling**
mp is working wiiii

missing:
 - auto leave lobby / game when loosing connection
 - notify other players about your death
 - opponents aren't picking up buffs
 - one of the highscore names doesn't show up
 - prettify GUI when exiting a game

### CODE REVIEW

File: app/src/main/java/team/adderall/fragments/LobbyFragment.java [hunk 1]
```diff
@@ -9,47 +9,81 @@
...
-// TODO: separate view and controller
+/**
+ * TODO: separate view and controller
+ */
 public class LobbyFragment
-        extends Fragment
+        extends
+        Fragment
 {
+    private final static String DEFAULT_USERNAME = "anonymous";
```
Comment (CimStordal) [inline app/src/main/java/team/adderall/fragments/LobbyFragment.java]: Maybe get from res/strings?

File: app/src/main/java/team/adderall/game/GameDetails.java [hunk 2]
```diff
@@ -103,7 +104,7 @@ public Intent writeToIntent(Intent intent) {
     }
 
     public long getHighscore() {
-        for(Player p: players.values()){
+        for(Player p : players){
```
Comment (CimStordal) [inline app/src/main/java/team/adderall/game/GameDetails.java]: Now we can use players.getActive() as players is no longer a map

File: app/src/main/java/team/adderall/game/ball/BallManager.java [hunk 3]
```diff
@@ -82,9 +82,9 @@ public BallManager() {
     /**
      * Constructor
      */
-    public BallManager(final boolean activePlayer) {
-
-        this.ball = new Ball(RADIUS);
+    public BallManager(final boolean activePlayer)
+    {
+        this.ball = new Ball(RADIUS, activePlayer ? "#F07030" : "#7070F0");
```
Comment (CimStordal) [inline app/src/main/java/team/adderall/game/ball/BallManager.java]: I guess we should also move these colors somewhere...

File: app/src/main/java/team/adderall/game/framework/GraphicsManager.java [hunk 4]
```diff
@@ -19,7 +19,7 @@
         implements
         GamePainter
 {
-    private static final double SCROLLSPEED = 1;
+    private static final double SCROLLSPEED = 0;
```
Comment (CimStordal) [inline app/src/main/java/team/adderall/game/framework/GraphicsManager.java]: Should still be 1

File: app/src/main/java/team/adderall/game/ball/BallManager.java (hunk 3 above)
Comment (andersfylling) [reply app/src/main/java/team/adderall/game/ball/BallManager.java]: fixed

File: app/src/main/java/team/adderall/game/framework/GraphicsManager.java (hunk 4 above)
Comment (andersfylling) [reply app/src/main/java/team/adderall/game/framework/GraphicsManager.java]: fixed

**Pull request merged**